test/parallel/test-buffer-readuintbe.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+// testing basic functionality of readUIntBE()
+
+const buf = Buffer.from([42, 84, 168, 127]);
+const result = buf.readUIntBE(2);
+
+assert.strictEqual(result, 84);
+
+assert.throws(
+   () => {
+     buf.readUIntBE(5);
+   }, RangeError
```

Comment:
This should follow the same recommendations @jasnell did here: https://github.com/nodejs/node/pull/10359#pullrequestreview-14269778